TelegramBot.py

```diff
@@ -1,13 +1,13 @@
 # Server Monitoring Bot
-import socket
-import json
 import argparse
+import json
+import socket
 import logging
-import time
 import datetime
```

Comment:
From the [ci output](https://travis-ci.org/Feathercoin-Foundation/Server-Monitoring-Bot/jobs/428988887#L471):
> ./TelegramBot.py:5:1: I100 Import statements are in the wrong order. 'import logging' should be before 'import socket'
./TelegramBot.py:6:1: I100 Import statements are in the wrong order. 'import datetime' should be before 'import logging'

Hint: the sorting needs to be in alphabetical order. Just swapping the import statements as suggested by the error message is not sufficient to achieve that. The message just gives an indicator why it found that the order to be wrong.